### PR TITLE
add function to provider to refetch data of sites

### PR DESCRIPTION
**Pull request by egrojMonroy**
recall data on delete polygon to reload

### CODE REVIEW

**Review by cesarLima1 — APPROVED** [review]

File: src/admin/components/ResourceTabs/PolygonReviewTab/components/Polygons.tsx [hunk 1]
```diff
@@ -101,8 +106,11 @@ const Polygons = (props: IPolygonProps) => {
   const deletePolygon = async (polygon: IPolygonItem) => {
     const response: any = await fetchDeleteV2TerrafundPolygonUuid({ pathParams: { uuid: polygon.uuid } });
     if (response && response?.uuid) {
-      const newMenu = polygonMenu.filter(item => item.uuid !== polygon.uuid);
-      setPolygonMenu(newMenu);
+      // const newMenu = polygonMenu.filter(item => item.uuid !== polygon.uuid);
```
Comment (cesarLima1) [inline src/admin/components/ResourceTabs/PolygonReviewTab/components/Polygons.tsx:109]: remove comments

File: src/components/elements/Map-mapbox/Map.tsx [hunk 2]
```diff
@@ -85,8 +85,12 @@ export const Map = ({
   const [viewImages, setViewImages] = useState(false);
   const { polygonsData, bbox, setPolygonFromMap, polygonFromMap } = props;
   const mapId = useId();
-  const sitePolygonData = useSitePolygonData();
+  const context = useSitePolygonData();
+  const sitePolygonData = context?.sitePolygonData;
   const refresh = useRefresh();
+  useEffect(() => {
+    console.log("site Po", sitePolygonData);
```
Comment (cesarLima1) [inline src/components/elements/Map-mapbox/Map.tsx:92]: removve console log

File: src/admin/components/ResourceTabs/PolygonReviewTab/components/Polygons.tsx (hunk 1 above)
Comment (egrojMonroy) [reply src/admin/components/ResourceTabs/PolygonReviewTab/components/Polygons.tsx:109]: done

File: src/components/elements/Map-mapbox/Map.tsx (hunk 2 above)
Comment (egrojMonroy) [reply src/components/elements/Map-mapbox/Map.tsx:92]: removed

**Pull request merged**